crates/oxc_regular_expression/src/display.rs

```diff
@@ -0,0 +1,586 @@
+use std::{
+    fmt::{self, Display},
+    iter::Peekable,
+};
+
+#[allow(clippy::wildcard_imports)]
+use crate::ast::*;
+use crate::body_parser::unicode::{combine_surrogate_pair, is_lead_surrogate, is_trail_surrogate};
```

Comment:
I'd like to split these 3 into `src/surroage-pair.rs` to normalize dependency graph. (prevent parent > child direction)